src/presentation/test/integration/registration/check-your-answers.test.ts

```diff
@@ -247,31 +253,33 @@ describe("Check Your Answers Page", () => {
     const limitedPartnerPerson = new LimitedPartnerBuilder().isPerson().withFormerNames("Joe Dee")
       .withContributionCurrencyType("GBP")
       .withContributionCurrencyValue("5.00")
-      .withContributionSubtypes(["MONEY", "LAND_OR_PROPERTY"])
+      .withContributionSubtypes(["MONEY", "SERVICES_OR_GOODS", "LAND_OR_PROPERTY"])
       .build();
 
     appDevDependencies.limitedPartnerGateway.feedLimitedPartners([limitedPartnerPerson]);
 
     const res = await request(app).get(URL);
 
     expect(res.status).toBe(200);
-    expect(res.text).toContain("5.00 Pound Sterling (GBP) Money / Land or property");
+    expect(res.text).toContain("5.00 Pound Sterling (GBP)");
+    expect(res.text).toContain("Money / Services or goods / Land or property");
   });
 
   it("should load the check your answers page with capital contribution data for limited partner legal entity", async () => {
 
     const limitedPartnerLegalEntity = new LimitedPartnerBuilder().isLegalEntity()
       .withContributionCurrencyType("GBP")
       .withContributionCurrencyValue("5.00")
-      .withContributionSubtypes(["MONEY", "LAND_OR_PROPERTY"])
+      .withContributionSubtypes(["SHARES", "ANY_OTHER_ASSET"])
       .build();
 
     appDevDependencies.limitedPartnerGateway.feedLimitedPartners([limitedPartnerLegalEntity]);
 
     const res = await request(app).get(URL);
 
     expect(res.status).toBe(200);
-    expect(res.text).toContain("5.00 Pound Sterling (GBP) Money / Land or property");
+    expect(res.text).toContain("5.00 Pound Sterling (GBP)");
+    expect(res.text).toContain("Shares / Any other asset");
```

Comment:
Are you able to maybe change this test to run using the Welsh locale and then check for the Welsh translations?

E.g. 

`expect(res.text).toContain("WELSH - Shares / WELSH - Any other asset");`

Reply:
Add a new test to check for welsh translations.